client/tailscale/localclient.go

```diff
@@ -846,6 +846,14 @@ func (lc *LocalClient) NetworkLockSign(ctx context.Context, nodeKey key.NodePubl
 	return nil
 }
 
+// NetworkLockDisable shuts down network-lock across the tailnet.
+func (lc *LocalClient) NetworkLockDisable(ctx context.Context, secret []byte) error {
+	if _, err := lc.send(ctx, "POST", "/localapi/v0/tka/disable", 200, bytes.NewReader(secret)); err != nil {
```

Comment:
Under the hood this all plumbs to `DoLocalRequest`, which sets nor checks none of the content types for any of the domain-socket RPCs. I presume we can avoid dealing with content type because this is a private channel where both ends are handled by us.